praetorian_cli/sdk/entities/jobs.py

```
@@ -316,6 +317,52 @@ def is_passed(self, job):
+
+        Returns:
+            Final job dict
+
+        Raises:
+            TimeoutError: If job doesn't complete within timeout
+        """
+        start = time.time()
+        while time.time() - start < timeout:
+            try:
+                job = self.api.search.by_exact_key(job_key)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)